### PR TITLE
Lukas first iteration userflow

### PR DESCRIPTION
**Pull request by lgerhardt45**
I implemented the user flow of our first iteration. 

**What works:**
- Videos/ video camera are used by default
- method stubs were added for @BergmuellerNiklas to add the PoseNet analysis on top of the frames from the videos
- all frames (seems like one per second) are shown in a scrollview for fast development
- all necessary buttons are there, but not necessarily functional

**What doesn't work properly yet:**
- navigation bar is some sort of magic in SwiftUI
- going back to the home screen from the ResultView
- showing the frames the smart way (image with a choose-image-bar below)

### CODE REVIEW

File: Jointify/ContentView.swift [hunk 1]
```diff
@@ -1,38 +0,0 @@
-//
-//  ContentView.swift
```
Comment (annalena94) [inline Jointify/ContentView.swift:2]: wtf

File: Jointify/Info.plist [hunk 2]
```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>By giving microphone access, you can give feedback to the doctor during the video</string>
+	<key>NSCameraUsageDescription</key>
+	<string>We need camera access to analyse your measurement</string>
 	<key>NSPhotoLibraryUsageDescription</key>
-	<string>Give me the cam pls</string>
+	<string>We need camera access to analyse your measurement</string>
```
Comment (annalena94) [inline Jointify/Info.plist:10]: I would call this "camera access is one of the fundamentals of this app"

Comment (annalena94) [reply Jointify/Info.plist:10]:
```suggestion
	<string>camera access is one of the fundamentals of this app</string>
```

Comment (lgerhardt45) [reply Jointify/Info.plist:10]:
```suggestion
	<string>Camera access is one of the fundamentals of this app</string>
```
A capital C should be good.

**Pull request merged**